ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -477,6 +477,24 @@ FAN_EFHLC_COCDSP_NOCDSP:
   - COCDSP
   - NOCDSP
 
+FAN_EFVSC_SCRSS:
```

Comment:
EF is not allowed on FAN types. Whether it is supply or exhaust is irrelevant to the type, that will be captured in the device name (e.g. EF-1). We only add the exhaust fan info when it differentiates it from another fan on the same device, and since this is a stand alone fan, differentiation is unnecessary.

Reply:
Updated